<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QCheckBox" name="enabledB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Włącz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QComboBox" name="devB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>83</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>63</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz urządzenie wyjściowe:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>113</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz urządzenie nagrywające:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="devNB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>133</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="detectB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Wykyj urządzenia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyDB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zastosuj wybrane urządzenia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="defDevB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Domyślne urządzenia</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="zB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Zawsze używaj tych urządzeń jeżeli istnieją</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Opóźnienie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Opóźnienie dla nagrywania:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lNB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoSurroundB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatycznie zmień urządzenie dźwiękowe &quot;default&quot; na
&quot;surround&quot; podczas odtwarzania dźwięku wielokanałowego</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>detectB</tabstop>
  <tabstop>devB</tabstop>
  <tabstop>devNB</tabstop>
  <tabstop>defDevB</tabstop>
  <tabstop>lB</tabstop>
  <tabstop>lNB</tabstop>
  <tabstop>zB</tabstop>
  <tabstop>autoSurroundB</tabstop>
  <tabstop>ApplyB</tabstop>
  <tabstop>ApplyDB</tabstop>
  <tabstop>enabledB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
